doc/codi.txt

```diff
@@ -246,6 +246,11 @@ g:codi#autocmd
                                                                 *g:codi#width*
 g:codi#width
              The width of the Codi split.
+             It can be a |Number|, a |Float| or a percentage string such as
+             `'33%'` or `'33.33%'`. The width will be clamped between `0.0`
+             and `100.0`. This plugin respects the |minwidth| variable if it
+             is present, otherwise the minimum remaining width is 10 columns.
```

Comment:
Aye, am already noticing that the fixed numbers are not handy to put in there. Since you also mentioned that this setting is built in and that it will always be present, might as well refer to the variable and be done with it.